src/package/versionNumber.ts

```diff
@@ -76,10 +77,16 @@ export class VersionNumber {
     }
   }
 
+  /**
+   * @deprecated use isBuildKeyword instead
+   */
   public isbuildKeyword(): boolean {
-    return Object.values(BuildNumberToken)
-      .map((v) => v.toString())
-      .includes(typeof this.build === 'string' && this.build.toUpperCase());
+    return this.isBuildKeyword();
+  }
+
+  public isBuildKeyword(): boolean {
+    const buildNumberTokenKeys = Object.values(BuildNumberToken);
+    return buildNumberTokenKeys.includes(this.build as BuildNumberToken);
```

Comment:
is this a place where build needs to be a number OR BuildNumberToken, or can it be any other string, too?